packages/core/src/tracing/instrumentation/protocols/httpServer.js

```diff
@@ -106,12 +106,6 @@ function shimEmit(realEmit) {
         header: getExtraHeadersFromMessage(req, extraHttpHeadersToCapture)
       };
 
-      const incomingServiceName =
```

Comment:
qs:

From the docs link you shared:
> By setting the X-Instana-Service HTTP header on a call, the destination (the service, which receives the HTTP request) is tagged with the value that is provided in the header.

How does it work when the tracers do not forward the header?

Reply:
The backend [code](https://github.ibm.com/instana/backend/blob/2a42269ff0d2608a843c0271375d45cd58f8d2a4/label-extractor/src/main/resources/defaultServiceRules.yaml#L37) reads the service name directly from the header, so we don't need to read X-Instana-Service header in our end.

Reply:
The customer have an option to configure the header in agent configuration file. Details explained in the PR description.